MovieApp/MovieApp/App/Features/MovieBrowser/MoviesViewController/MovieDetail/MovieDetailViewController.swift

```diff
@@ -61,7 +80,7 @@ extension MovieDetailViewController: MovieDetailDelegate {
     }
     
     func reloadData() {
-        titleView.reloadData()
+        //titleView.reloadData()
     }
```

Comment:
no need for this anymore

Reply:
[Deleted](https://github.com/lihter/Movie-App/pull/47/files#diff-f5b01144bf2cdd8175add69b29bdfdcf3abf149a38fdd4929f07da30b107f914L82-L84)